x-pack/plugins/secops/public/store/local/hosts/actions.ts

```diff
@@ -19,3 +20,15 @@ export const updateEventsLimit = actionCreator<{ limit: number }>('UPDATE_EVENTS
 export const updateUncommonProcessesLimit = actionCreator<{ limit: number }>(
   'UPDATE_UNCOMMONPROCESSES_LIMIT'
 );
+
+export const updateUncommonProcessesUpperLimit = actionCreator<{ upperLimit: number }>(
```

Comment:
`upperLimit` is no longer with us and not needed. So this is  💀  dead code.

Reply:
Bad rebasing!!!